### PR TITLE
Adding geoapis

### PR DESCRIPTION
**Pull request by rosepearson**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by rosepearson** [comment]
@conda-forge/help-python ready for review

I am new to publishing Python packages so their are a few things I'm not sure about. I have a dependency on GDAL (through geopandas) which means I have trouble installing with pip. Please let me know if I should be taking a different approach! I'm very keen to take the recommended approach. Thanks!

**Comment by rosepearson** [comment]
Added a second package to publish that is dependent on the first as well as one other package not on conda-forge. As a results the tests now fail. Keen to get some guidance when you triage tomorrow!

**Comment by BastianZim** [comment]
Hi @rosepearson I would recommend basing it off a GitHub release in your case and handling the dependency management with conda. Just make sure to only include your dependencies. Dependencies needed by those will then be pulled in by conda.
There are many packages that don't publish to PyPI so this is perfectly fine to do.

**Comment by rosepearson** [comment]
Hi @BastianZim thanks for the comment. I've been on holiday - back to it now though. I'm just reading up on how to do that now - I didn't find an obvious example that used a GitHub release initially. Do you have any example recipes you could easily point me too? I'll update here as I progress.

**Comment by rosepearson** [comment]
@BastianZim I have all tests passing. Could this please be reviewed now.

File: recipes/geoapis/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  host:
+    - pip
+    - python
+  run:
+    - boto3
+    - geopandas >=0.10
+    - python
+    - python-dotenv
+    - requests
+    - tomli  # [py<311]
```
[the remark this answers is not in the document]
Comment (rosepearson) [reply recipes/geoapis/meta.yaml:27]: Give python version upper limit

**Comment by rosepearson** [comment]
Thanks @ngam Your changes look good to me. I've accepted them! What's next for the review process?
@conda-forge/help-python

**Pull request merged**